src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -327,16 +323,16 @@
              ;; method redact is responding with "code" for 401.  TODO: standardize and remove this extra logic
              status-code (or status-code code)]
          (if-let [expected-msg (get-in props [:expect status-code])]
-           [:div {}
+           [:div {:style {:display "flex" :alignItems "center"}}
```

Comment:
A lot of little alignment things popped up when I changed the icons.  This is more robust anyway.